### PR TITLE
[luci-interpreter] Test quantized Pad kernels for ResNet-v1

### PR DESCRIPTION
**Pull request by struss**
Parent Issue: #2730 
Draft PR: #2720

This commit update `Pad` operator test case.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

**Comment by struss** [comment]
Some Example value is different from float case, so I just add new uint8 case.

File: compiler/luci-interpreter/src/kernels/Pad.test.cpp [hunk 1]
```diff
@@ -26,6 +26,34 @@ namespace
 
 using namespace testing;
 
+// for quantized Add, the error shouldn't exceed step
+float GetTolerance(int min, int max) { return (max - min) / 255.0; }
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Pad.test.cpp:30]: min, max should be float.

please see https://github.com/Samsung/ONE/pull/2800/files#r448751543.

Comment (struss) [reply compiler/luci-interpreter/src/kernels/Pad.test.cpp:30]: Oh sorry! I will fix it!

**Comment by struss** [comment]
@nnfw-bot

**Comment by struss** [comment]
@nnfw-bot test

**Comment by struss** [comment]
@nnfw-bot test format-checker

**Pull request merged**